packages/manager/modules/telecom-dashboard/src/identity-check/form/telecom-dashboard-identity-check-form.html

```diff
@@ -0,0 +1,178 @@
+<section class="telecom-legacy">
+    <tuc-toast-message></tuc-toast-message>
+    <div class="text-center" data-ng-if="$ctrl.isLoading">
+        <oui-spinner></oui-spinner>
+    </div>
+    <div data-ng-if="!$ctrl.isLoading">
+        <div class="telecom-dashboard">
+            <header class="oui-header mb-4">
```

Comment:
Component doesn't suit the need ?

Reply:
There is data-ng-if on description. oui-header does not allow the condition to be changed once the text has been generated.